client/state/plugins/wporg/actions.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+const debug = require( 'debug' )( 'calypso:wporg-data:actions' );
+
+/**
+ * Internal dependencies
+ */
+import wporg from 'lib/wporg';
+import utils from 'lib/plugins/utils';
+import { WPORG_PLUGIN_DATA_RECEIVE, FETCH_WPORG_PLUGIN_DATA } from 'state/action-types';
+
+/**
+ *  Local variables;
+ */
+let _fetching = {};
+
+export default {
+	fetchPluginData: function( pluginSlug ) {
+		return ( dispatch ) => {
+			if ( _fetching[ pluginSlug ] ) {
```

Comment:
While this was pretty common in Flux action creators, I think we should avoid local variables as much as possible, since these are arguably part of the application state. Instead, you might consider including the fetching progress in your reducer, and creating a selector if you need to know the current fetching progress (preferably from whatever invokes `fetchPluginData`, not from within the action creator itself).